### ISSUE TITLE
The button doesn't work and Auto select text

### ISSUE THREAD

**Issue by taolive** [kingToolbox/WindTerm#138]
1、Sometimes a button needs to be clicked multiple times
2、Moving the mouse automatically selects the text block

os：macOS monterey beta3
pc：MacBook Pro (13-inch, 2019, Four Thunderbolt 3 ports)
version：WindTerm 2.1.0

**Comment by kingToolbox** [comment]
Yes, I encountered the second problem a long time ago, but because it happens occasionally, I haven't troubleshooted it. I will fix it later. And for the first question, which button, or which buttons, can you give a specific example? Thank you.

**Comment by taolive** [comment]
> Yes, I encountered the second problem a long time ago, but because it happens occasionally, I haven't troubleshooted it. I will fix it later. And for the first question, which button, or which buttons, can you give a specific example? Thank you.

<img width="217" alt="iShot2021-07-28 21 50 51" src="https://user-images.githubusercontent.com/1654524/127334005-537b4b97-4640-4717-a62b-c0ac4641583a.png">
<img width="678" alt="iShot2021-07-28 21 50 33" src="https://user-images.githubusercontent.com/1654524/127334049-23272452-fe69-46d0-8978-8a08f6a4874e.png">


Sometimes clicking the same button many times, sometimes clicking once has an effect

**Comment by kingToolbox** [comment]
Okay, I will investigate this issue. If there is any progress, I will update it here. Thank you.

**Comment by kingToolbox** [comment]
The new [WindTerm_2.2.0_Prerelease_4](https://github.com/kingToolbox/WindTerm/releases/tag/2.2-prerelease) version has been released and the text will not be automatically selected by mistake when moving the mouse, please download and check it, thank you.

BTW:
I have never been able to reproduce the bug that the buttons may be clicked multiple times to trigger. Please check if the same error still exists in the new `WindTerm_2.2.0_Prerelease_4` version. If it still exists, please let me know and I will continue to try to fix it.
